libdnf/dnf-repo.c

```diff
@@ -787,6 +898,8 @@ dnf_repo_set_keyfile_data(DnfRepo *repo, GError **error)
 {
     DnfRepoPrivate *priv = GET_PRIVATE(repo);
     guint cost;
+    guint metadata_expire;
+    g_autofree gchar *metadata_expire_str;
```

Comment:
Add ` = NULL` here like the others; without that the memory isn't initialized which is dangerous with autocleanups.